mne/utils/mixin.py

```diff
@@ -495,6 +495,15 @@ def crop(self, tmin=None, tmax=None, include_tmax=True, verbose=None):
         -----
         %(notes_tmax_included_by_default)s
         """
+        t_vars = dict(tmin=tmin, tmax=tmax)
+        for name, t_var in t_vars.items():
+            _validate_type(
+                t_var,
+                types=("numeric", None),
+                item_name=name,
+                type_name="int, float, None",
```

Comment:
I don't think you should need this type_name, we should live with however the "numeric" gets printed (or improve it)

Reply:
I checked before implementing anything myself how it was done for `_handle_tmin_tmax` again under `TimeMixin` and this is the code used there (formatting may vary due to black):

```
        _validate_type(
            tmin,
            types=("numeric", None),
            item_name="tmin",
            type_name="int, float, None",
        )
```

However, as requested, I won't provide a `type_name` optional parameter.

I will also add the requested line to `changes/latest.inc` :)

Reply:
I added myself as `:newcontrib:` since it's the first time I contribute code, but in theory I have opened issues for MNE in the past.